system-tests/__snapshots__/vite_dev_server_fresh_spec.ts.js

```diff
@@ -1,850 +1,4 @@
-exports['@cypress/vite-dev-server react executes all of the tests for vite3.0.2-react 1'] = `
```

Comment:
This deletion doesn't feel right - I would expect a large addition here for 4.0.4

Reply:
You're right, looks like I committed the wrong updated snapshot, will come back to this after some IATR stuff.